protocol/communities/manager.go

```diff
@@ -76,7 +76,7 @@ type Manager struct {
 	walletConfig                   *params.WalletConfig
 	historyArchiveTasksWaitGroup   sync.WaitGroup
 	historyArchiveTasks            map[string]chan struct{}
-	periodicMemberPermissionsTasks map[string]chan struct{}
+	periodicMemberPermissionsTasks sync.Map
```

Comment:
Since we're losing type information here, maybe add a commit behind `sync.Map` that this stores `chan struct{}`?